packages/components/src/JSONSchemaRenderer/JSONSchemaRenderer.test.js

```diff
@@ -218,17 +210,12 @@ describe('JSONSchemaRenderer', () => {
 		};
 		const wrapper = mount(<JSONSchemaRenderer schema={schema} />);
 		expect(wrapper.find('dt')).toHaveLength(1);
-		expect(
-			wrapper
-				.find('dt')
-				.first()
-				.text(),
-		).toBe('a');
+		expect(wrapper.find('dt').first().text()).toBe('a');
 	});
 
 	it('should throw an exception in case of invalid schema', () => {
 		const wrapper = () => shallow(<JSONSchemaRenderer schema={{}} />);
-		expect(wrapper).toThrow(InvalidSchemaException);
+		expect(wrapper).toThrow('Invalid Schema');
```

Comment:
`expect(wrapper).toThrow(InvalidSchemaException);` was equivalent to `expect(wrapper).toThrow();` because the error is imported but not exposed, so it's undefined.

I changed the assertion to match the expected message